### PR TITLE
Update risk-score.md

### PR DESCRIPTION
**Pull request by rajesh-k123**

### CODE REVIEW

**Comment by dcurwin** [comment]
Learn Build status updates of commit _[ceb6c74](https://github.com/rajesh-k123/CloudAppSecurityDocs/commits/ceb6c74cf6644ae28006433c1bfe3b40bffeaff9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[CloudAppSecurityDocs/risk-score.md](https://github.com/rajesh-k123/CloudAppSecurityDocs/blob/patch-1/CloudAppSecurityDocs/risk-score.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/daf9759b-aa27-219c-4b5e-53f1b72a2d34/202211091125150601-607/BuildReport?accessString=8743ed9632ef2f35ce2acd1eb339f1410e4faacfa77ec18a84071aeb21cd834e).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/daf9759b-aa27-219c-4b5e-53f1b72a2d34?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

File: CloudAppSecurityDocs/risk-score.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ There are basic and advanced Cloud App Catalog filters. To build a complex filte
 - **Apps and domains**: Enables you to search for specific apps or apps used in specific domains.
 - **Categories**: The categories filter, which is located on the left of the page, enables you to search for types of apps according to app categories. For example Social network apps, Cloud storage apps, and more types of apps. You can select more than one category at a time or a single category. Then, apply the basic or advanced filters on top of the categories.
 - **Compliance risk factor**: Lets you search for a specific standards, certification, and compliance that the app may comply with. Examples include HIPAA, ISO 27001, SOC 2, and PCI-DSS.
-- **General risk factor**: Lets you search for general risk factors such as consumer popularity, data center locale, and more.
+- **General risk factor**: Lets you search for general risk factors such as Headquarters, data center locale, and more.
```
Comment (dcurwin) [inline CloudAppSecurityDocs/risk-score.md:47]:
```suggestion
- **General risk factor**: Lets you search for general risk factors such as headquarters, data center locale, and more.
```

File: CloudAppSecurityDocs/risk-score.md [hunk 2]
```diff
@@ -79,17 +79,17 @@ If you find a new app in your environment that hasn't been scored by Defender fo
...
+Each of the categories is composed of many specific properties. Few properties are not considered for score calculation and are presented for information only. According to the Defender for Cloud Apps scoring algorithm, each property (that is considered for score calculation) receives a preliminary score between 0 and 10, depending on the value. True/False values will receive 10 or 0 accordingly. However, continuous properties, such as domain age, will receive a certain value within the spectrum. The score of each property is weighted against all other existing fields in the category to create the category's subscore. If you encounter an unscored app, it usually indicates an app whose properties are unknown and is therefore unscored.
```
Comment (dcurwin) [inline CloudAppSecurityDocs/risk-score.md:92]:
```suggestion
Each of the categories is composed of many specific properties. A few properties are not considered for score calculation and are presented for information only. According to the Defender for Cloud Apps scoring algorithm, each property (that is considered for score calculation) receives a preliminary score between 0 and 10, depending on the value. True/False values will receive 10 or 0 accordingly. However, continuous properties, such as domain age, will receive a certain value within the spectrum. The score of each property is weighted against all other existing fields in the category to create the category's subscore. If you encounter an unscored app, it usually indicates an app whose properties are unknown and is therefore unscored.
```

File: CloudAppSecurityDocs/risk-score.md [hunk 3]
```diff
@@ -79,17 +79,17 @@ If you find a new app in your environment that hasn't been scored by Defender fo
 
 ## Customizing the risk score
 
-Cloud Discovery provides you with important data regarding the credibility and reliability of the cloud apps that are used across the environment. Within the portal, each discovered app is displayed along with a total score. The score represents the Defender for Cloud Apps assessment of this particular app's maturity of use for enterprises. The total score of any given app is a weighted average of four subscores relating to the four subcategories which Defender for Cloud Apps considers when assessing reliability:
+Cloud Discovery provides you with important data regarding the credibility and reliability of the cloud apps that are used across the environment. Within the portal, each discovered app is displayed along with a total score. The score represents the Defender for Cloud Apps assessment of this particular app's maturity of use for enterprises. The total score of any given app is a weighted average of three subscores relating to the three subcategories - Security, Compliance and Legal, which Defender for Cloud Apps considers when assessing reliability:
```
Comment (dcurwin) [inline CloudAppSecurityDocs/risk-score.md:82]:
```suggestion
Cloud Discovery provides you with important data regarding the credibility and reliability of the cloud apps that are used across the environment. Within the portal, each discovered app is displayed along with a total score. The score represents the Defender for Cloud Apps assessment of this particular app's maturity of use for enterprises. The total score of any given app is a weighted average of three subscores relating to the three subcategories (Security, Compliance and Legal) which Defender for Cloud Apps considers when assessing reliability:
```

**Review by dcurwin — APPROVED** [review]

**Comment by dcurwin** [comment]
The PR was approved. When you're ready to publish, please type `#sign-off` in a new comment box, and click **Comment**.

**Comment by Court72** [comment]
@dcurwin,

Because this PR was opened in the public repo, the article's author must sign off. Since [you've approved the changes](https://github.com/MicrosoftDocs/CloudAppSecurityDocs/pull/607#issuecomment-1308636262), we'll go ahead and merge. Thanks!

**Comment by Court72** [comment]
@dcurwin

Because this PR was opened in the public repo, only the article's author can sign off. Since you've [approved the changes](https://github.com/MicrosoftDocs/CloudAppSecurityDocs/pull/607#issuecomment-1308636262), we'll go ahead and merge. Thanks!

**Comment by dcurwin** [comment]
@Court72  - Can you please revert this PR as well? It should not have been published yet.

**Comment by dcurwin** [comment]
> @Court72 - Can you please revert this PR as well? It should not have been published yet.

Actually, I was able to revert it myself.

**Pull request merged**